scripts/results/session-nack.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+#.....................................................................
+# Copyright (c) 2016-2019, Regents of the University of Arizona.
+# Author: Wenkai Zheng <wenkaizheng@email.arizona.edu>
```

Comment:
Minor comment, if you borrow code from anywhere you should give the source credit and reference (https://ohiostate.pressbooks.pub/choosingsources/chapter/ethical-use-and-citing-sources/).